src/Domain/Insights/ForbiddenSecurityIssues.php

```diff
@@ -70,7 +70,8 @@ private function getResult(): Result
 
             try {
                 self::$result = $checker->check(sprintf(
-                    '%s/composer.lock', $this->collector->getDir()
+                    '%scomposer.lock',
```

Comment:
Does commonPath include the backslash or did you forget to add it in here?

Reply:
Yes it contains it always.